packages/scene-composer/src/components/panels/scene-components/data-overlay/DataBindingMapNameEditor.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useCallback } from 'react';
+import { FormField, Input } from '@awsui/components-react';
+import { useIntl } from 'react-intl';
+import { isEmpty } from 'lodash';
+
+import { IDataOverlayComponentInternal } from '../../../../store';
+import { Component } from '../../../../models/SceneModels';
+
+interface IDataBindingMapNameEditorProps {
+  bindingName: string;
+  index: number;
+  valueDataBindings: Component.ValueDataBindingNamedMap[];
+  onUpdateCallback: (componentPartial: Partial<IDataOverlayComponentInternal>, replace?: boolean | undefined) => void;
+}
+
+export const DataBindingMapNameEditor: React.FC<IDataBindingMapNameEditorProps> = ({
+  bindingName,
+  index,
+  valueDataBindings,
+  onUpdateCallback,
+}) => {
+  const { formatMessage } = useIntl();
+
+  const bindingNameError = useCallback(
+    (bindingName) => {
+      if (isEmpty(bindingName)) {
+        return formatMessage({ defaultMessage: 'Invalid name', description: 'Input error message' });
+      }
+      if (valueDataBindings.filter((v) => v.bindingName === bindingName).length > 1) {
+        return formatMessage({ defaultMessage: 'Duplicate name', description: 'Input error message' });
+      }
+      return null;
+    },
+    [valueDataBindings, formatMessage],
+  );
+
+  const onBindingNameChange = useCallback(
+    (e, index) => {
+      const newBindings = [...valueDataBindings];
+      newBindings[index] = { ...newBindings[index], bindingName: e.detail.value };
+      onUpdateCallback({ valueDataBindings: newBindings });
+    },
+    [valueDataBindings, onUpdateCallback],
+  );
+
+  return (
+    <FormField
+      errorText={bindingNameError(bindingName)}
+      label={formatMessage({ defaultMessage: 'Binding Name', description: 'FormField label' })}
```

Comment:
Should not this be `Binding name`?

Reply:
this is to be consistent with the other labels for data binding, like "Entity Id", "Component Name" (see the screenshot in description). I guess the "Markdown content" should follow the same pattern.. I will update it in next PR since I will update that component again for next task